### DISCUSSION TITLE
rtl issues

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [robertodoering/harpy#743] [Ideas]
in rtl shouldnt username follow the left to right direction(original client follows it like that)?
![Screenshot_2022-08-17-22-34-24-657_com.robertodoering.harpy.free.jpg](https://user-images.githubusercontent.com/38470944/185205449-eea48f0b-945d-4cdb-9e8d-066a2a054da9.jpg)

**Comment by robertodoering** [comment]
I don't know, should it? Is that how it's displayed in the official Twitter client?
Input from a native speaker would be helpful
cc @FerasK0

**Reply by FerasK0** [reply]
I don't see anything wrong honestly except maybe the @ placement which is inverted in a RTL context right now.

And actually after checking it out, it appears that if there's more than one mention in a tweet it kinda bugs out.

I guess what would be an effective fix is to make LTR based hashtags and mentions Always LTR (#harpy -  @harpy), and since there's no RTL mentions, keep the RTL Hashtags the they're right now. (#هاربي -  check screenshot below)


Here's two Screenshots which shows the obscure @ behavior. 
![Screenshot_20220818-005303_harpy.png](https://user-images.githubusercontent.com/83197825/185250395-0e399122-30f6-4594-bb94-fcdf5a92b8be.png)

![Screenshot_20220818-005240_harpy.png](https://user-images.githubusercontent.com/83197825/185250405-96df41ab-e4d0-4d96-95b8-06fa5865e400.png)


And here's the RTL hashtags "keep it like that :3"

![Screenshot_20220818-005843_harpy.png](https://user-images.githubusercontent.com/83197825/185250806-754d9624-93b5-466c-bec9-f931ef632aaa.png)

**Reply by FerasK0** [reply]
I also forgot to add something which would be a feature that would fix the mentions situation.

![Screenshot_20220818-010417_Twitter.png](https://user-images.githubusercontent.com/83197825/185251944-d6932379-d57d-42f7-b2cd-44ed19fbb27a.png)

Is it possible to make replies to mentioned ppl appear just like in Twitter? As if not in tweet itself but in separate entity if you get what I mean. 😅

**Reply by HikaruIchijyo** [reply]
> I don't know, should it? Is that how it's displayed in the official Twitter client?
> Input from a native speaker would be helpful
> cc @FerasK0

yeah hashtags & mentions should follow ltr approach
